frontend/src/dto/SimulationResults.tsx

```diff
@@ -1,27 +1,14 @@
+import { ModelConfig } from "./FormConfig";
+
+type JSONPrimitive = string | number | boolean | null;
+
+type JSONSerializable = JSONPrimitive | JSONSerializable[] | { [k: string]: JSONSerializable | undefined };
+
+type Visualisations = JSONSerializable | string;
+
 export interface SimulationResults {
-    results: {
-        initfinaldiff: {
-            initial: { [key: string]: number };
-            final: { [key: string]: number };
-            change: { [key: string]: number };
-        };
-    };
-    analysis: {
-        common_paths: {
-            paths: Record<string, string>;
-            k: Record<string, string>;
-            frequency: Record<string, number>;
-        };
-        stats: {
-            index: Record<string, string>;
-            k: Record<string, string>;
-            frequency: Record<string, number>;
-        };
-    };
-    chart_data: {
-        comps: { [key: string]: string };
-        values: { [key: string]: number };
-        variance: { [key: string]: number };
-        variance_minus: { [key: string]: number };
-    };
+    modelConfig: ModelConfig;
```

Comment:
Suggested to remove the `ModelConfig` from `RunResponse`, we might want to remove it from `SimulationResults` as well?

Although, we do want to know the name of the model.. And I would argue even more, like model version.. 

Likewise with suggested on the other sides, rename `visualizations` to something else.